Doc/pages/R_traj.rst

```diff
@@ -1,84 +1,84 @@
-|                   |                                      |                   | data                                 | Dedicated to X-PLOR format conversions,  | ensuring accurate trajectory                 |
-|                   |                                      |                   |                                      | ensuring accurate trajectory             | representation.                              |
-|                   |                                      |                   |                                      | representation.                          |                                              |
-+-------------------+--------------------------------------+-------------------+--------------------------------------+------------------------------------------+----------------------------------------------+
++-------------------+--------------------------------------+-------------------+------------------------------------------+----------------------------------------------+
+| Type              | File Format                          | Default Extension | Description                              | Improved Other Information                   |
++===================+======================================+===================+==========================================+==============================================+
+| CASTEP Converter  | CASTEP trajectory format             | .md               | Converts CASTEP trajectory files to HDF  | Supports all CASTEP file versions with       |
+|                   |                                      |                   | format, including velocities and forces. |  metadata integration.                       |
++-------------------+--------------------------------------+-------------------+------------------------------------------+----------------------------------------------+
+| CHARMM Converter  | CHARMM trajectory format             |  .pdb (PDB file), | Converts CHARMM trajectory files to HDF  | Integration with PDB file formats for        |
+|                   |                                      | other             | format. Integration with PDB file        | enhanced CHARMM trajectory processing.       |
+|                   |                                      |                   | formats for enhanced CHARMM trajectory   |                                              |
+|                   |                                      |                   | processing.                              |                                              |
++-------------------+--------------------------------------+-------------------+------------------------------------------+----------------------------------------------+
+| CP2K Converter    | CP2K trajectory format               |  .xyz (position), | Converts CP2K trajectory files to HDF    | Velocity approximation from positions        |
```

Comment:
Some formatting issues on my end.

![image](https://github.com/ISISNeutronMuon/MDANSE/assets/14276033/90701560-8ef9-4ec9-aefc-da3beb43e6de)

I think this is due to some extra spaces in table cells.